### ISSUE TITLE
Crash when I popup a video

### ISSUE THREAD

**Issue by silas229** [TeamNewPipe/NewPipe#7575]
## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ DE
* __Content Language:__ en-DE
* __App Language:__ en_DE
* __Service:__ none
* __Version:__ 0.21.14
* __OS:__ Linux Android 11 - 30
<details><summary><b>Crash log </b></summary><p>

```
java.lang.NullPointerException: Attempt to invoke virtual method 'org.schabi.newpipe.extractor.stream.StreamInfo org.schabi.newpipe.player.resolver.MediaSourceTag.getMetadata()' on a null object reference
	at org.schabi.newpipe.player.Player.onPlayerError(Player.java:2526)
	at com.google.android.exoplayer2.ExoPlayerImpl.lambda$updatePlaybackInfo$10(ExoPlayerImpl.java:1269)
	at com.google.android.exoplayer2.ExoPlayerImpl.$r8$lambda$-K5msfvAakBrxKAsG9AjoK30L1g(Unknown Source:0)
	at com.google.android.exoplayer2.ExoPlayerImpl$$ExternalSyntheticLambda7.invoke(Unknown Source:4)
	at com.google.android.exoplayer2.util.ListenerSet$ListenerHolder.invoke(ListenerSet.java:283)
	at com.google.android.exoplayer2.util.ListenerSet.lambda$queueEvent$0(ListenerSet.java:170)
	at com.google.android.exoplayer2.util.ListenerSet.$r8$lambda$NbKDn9xtItiyMgYZmjIx_Sv1FFQ(Unknown Source:0)
	at com.google.android.exoplayer2.util.ListenerSet$$ExternalSyntheticLambda1.run(Unknown Source:6)
	at com.google.android.exoplayer2.util.ListenerSet.flushEvents(ListenerSet.java:191)
	at com.google.android.exoplayer2.ExoPlayerImpl.updatePlaybackInfo(ExoPlayerImpl.java:1340)
	at com.google.android.exoplayer2.ExoPlayerImpl.handlePlaybackInfo(ExoPlayerImpl.java:1175)
	at com.google.android.exoplayer2.ExoPlayerImpl.lambda$new$1(ExoPlayerImpl.java:216)
	at com.google.android.exoplayer2.ExoPlayerImpl.$r8$lambda$nOBJYkeEQ2uz3sBKLToLWmzrgZk(Unknown Source:0)
	at com.google.android.exoplayer2.ExoPlayerImpl$$ExternalSyntheticLambda22.run(Unknown Source:4)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:264)
	at android.app.ActivityThread.main(ActivityThread.java:8335)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:632)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1049)

```
</details>
<hr>

**Comment by bolshoytoster** [comment]
The error was fixed in [this commit](https://github.com/TeamNewPipe/NewPipe/commit/c8e2ab4c83f93a3ce3561a30f53f584af40dce2e#diff-68cd2f051cf461f1cee8f97ab4a793ab90276943eed26f8f1baf5105948b2b5e).

**Comment by litetex** [comment]
Fixed by #7482
